README.md

```diff
@@ -1027,6 +1027,8 @@ Table of Contents
   * [GraphComment](https://graphcomment.com/) - GraphComment is a comments platform that helps you build an active community from website’s audience. 
   * [Utterances](https://utteranc.es/) - A lightweight comments widget built on GitHub issues. Use GitHub issues for blog comments, wiki pages and more!
   * [Disqus](https://disqus.com/) - Disqus is a networked community platform used by hundreds of thousands of sites all over the web.
+  * [Gitalk](https://gitalk.github.io/) - Gitalk is a modern comment component based on Github Issue and Preact. 
```

Comment:
I wouldnt say this is a *aaS, this is a library that uses github as a backend, it's nice but its not a *aaS.